src/parsers/expression/selection.js

```diff
@@ -99,14 +109,34 @@ function vlSelection(name, datum, op, unit, scope, test) {
 
 // Assumes point selection tuples are of the form:
 // {unit: string, encodings: array<string>, fields: array<string>, values: array<*>, bins: object}
-export function vlPoint(name, datum, op, unit, scope) {
-  return vlSelection.call(this, name, datum, op, unit, scope, testPoint);
+export function vlSingle(name, datum, op) {
+  return vlSelection.call(this, name, datum, op, testPoint);
+}
+
+export function vlMulti(name, datum, op) {
```

Comment:
Looks like this is identical to `vlSingle`. Why the redundant definition and export?